File: uri/C/beginner/1071.c
#include <stdio.h>

int main()
{

	int X, Y, i, maior, menor, num=0;

	scanf("%d %d", &X, &Y);

	if(X > Y){
		maior = X;
		menor = Y;
	}
	else{
		menor = X;
		maior = Y;
	}

	for(i = (menor + 1); i < maior; i++){
		if(i % 2 != 0){
			num += i;
		}
	}

	printf("%d\n", num);

	return 0;
}

File: uri/python/1071.py
# -*- coding: utf-8 -*-

primeiro = int(input())
segundo = int(input())
num = 0

if primeiro > segundo:
	maior = primeiro
	menor = segundo
else:
	maior = segundo
	menor = primeiro

aux = menor + 1

while aux < maior:
	if aux % 2 != 0:
		num += aux
	aux += 1

print('{}'.format(num))

